### PR TITLE
Remove inter-feature dependencies for 'wallet'

### PR DESCRIPTION
**Pull request by Tyratox**
This PR removes all inter-feature dependencies for the wallet feature with the exception of the `DummyWallet` since I assume that it is not planned that it stays there forever?

### CODE REVIEW

**Review by 0xdieman — APPROVED** [review]
LGTM, and yes the dummy wallet is not supposed to stay for very long

**Review by pierluca — COMMENTED** [review]
Minor remark, but LGTM. Good job !

File: fe1-web/src/features/wallet/screens/WalletHome.tsx [hunk 1]
```diff
@@ -34,40 +32,45 @@ const styles = StyleSheet.create({
...
       setTokens(undefined);
       return;
     }
 
-    Wallet.recoverWalletRollCallTokens(rollCalls, lao)
+    Wallet.recoverWalletRollCallTokens(rollCalls, laoId)
       .then((rct) => {
+        console.log('recoverWalletRollCallTokens.then', rct);
         if (rct.length > 0) {
+          console.log('call set tokens', rct);
```
[the remark this answers is not in the document]
Comment (Tyratox) [reply fe1-web/src/features/wallet/screens/WalletHome.tsx:65]: No I don't! Thank you for pointing them out!

**Review by pierluca — APPROVED** [review]
LGTM

**Pull request merged**